app/main.py

```diff
@@ -1,8 +1,40 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int, clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, dirty_cars: list[Car]) -> float:
+        total_income = 0
+        for dirty_car in dirty_cars:
+            if dirty_car.clean_mark < self.clean_power:
+                total_income += self.wash_single_car(dirty_car)
+        return round(total_income, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        income = (car.comfort_class
+                  * (self.clean_power - car.clean_mark)
+                  * self.average_rating
+                  / self.distance_from_city_center)
+        return round(income, 1)
+
+    def wash_single_car(self, car: Car) -> float:
+        price = self.calculate_washing_price(car)
```

Comment:
Pay attention to the name of the function, it does not involve any calculation here
![image](https://github.com/mate-academy/py-car-wash-station/assets/102595649/dff082ab-f5b7-4905-bc47-0cac5d61cd96)